packages/admin-stories/src/docs/components/Stack/Stack.stories.mdx

```diff
@@ -49,13 +79,48 @@ This method will activate (navigate to) the given page.
 
 #### Arguments
 
-1. `pageName` (string): name of the page that will be activated
-2. `payload` (string): additional payload containing eg. an id that will get passed into the StackPage
-3. `subUrl` (string [optional]): subUrl to append, for nested stacks or other nested routes
+| Name       | Type   | Default value | Description                                                                      |
+| :--------- | :----- | :------------ | :------------------------------------------------------------------------------- |
+| pageName\* | string |               | Name of the page that will be activated.                                         |
+| payload\*  | string |               | Additional payload containing eg. an id that will get passed into the StackPage. |
+| subUrl     | string |               | SubUrl to append, for nested stacks or other nested routes.                      |
 
 ### StackLink
 
-A StackLink component is planned that will work similarly to react-router Link.
+It's also possible to use the `StackLink` component for creating a link between pages:
+
+<Canvas>
+    <Story id="stories-components-stack--stacklink" />
+</Canvas>
+
+#### Props
+
+| Name       | Type            | Default value | Description                                                                                    |
+| :--------- | :-------------- | :------------ | :--------------------------------------------------------------------------------------------- |
+| pageName\* | string          |               | Name of the page that will be linked to.                                                       |
+| payload\*  | string          |               | Additional payload containing eg. an id that will get passed into the StackPage.               |
+| subUrl     | string          |               | SubUrl to append, for nested stacks or other nested routes.                                    |
+| switchApi  | IStackSwitchApi |               | Custom switchApi, has to be passed if the StackLink is used outside of a StackSwitch component |
+
+### getTargetUrl
+
+The `getTargetUrl()` method returns the url of a page in the stack:
+
+<Source
+    language="tsx"
+    code={dedent`
+        const switchApi = useStackSwitchApi();
+        const url = switchApi.getTargetUrl("page2", "test payload");
+  `}
+/>
+
+#### Arguments
+
+| Name       | Type   | Default value | Description                                                                      |
+| :--------- | :----- | :------------ | :------------------------------------------------------------------------------- |
+| pageName\* | string |               | Name of the page that will be activated.                                         |
+| payload\*  | string |               | Additional payload containing eg. an id that will get passed into the StackPage. |
+| subUrl     | string |               | SubUrl to append, for nested stacks or other nested routes.                      |
```

Comment:
I would remove the Default value column if there is no single default value in the table